watch_manager_error.go

```diff
@@ -0,0 +1,14 @@
+package nutsdb
```

Comment:
I think moving these error to watch_manager.go is better. We don't have to let errors in another file.